src/event.js

```diff
@@ -404,13 +404,13 @@ export const Events = {
     };
 
     var bind = opt.bind;
-
     if (bind) {
-      for (var name in opt) {
+      for (let name in opt) {
```

Comment:
const name in opt should be better.

Try 'npm run lint' before you commit, it should warn about this.

Also if you are using sublime you can install sublime-linter and it should highlight the linter errors inline in the code.